sections/futures/TradeCrossMargin/TradeCrossMargin.tsx

```diff
@@ -108,10 +108,10 @@ export default function TradeCrossMargin({ isMobile }: Props) {
 							const price =
 								(type === 'limit' && leverageSide === 'long') ||
 								(type === 'stop market' && leverageSide === 'short')
-									? floorNumber(marketAssetRate, 0)
+									? floorNumber(marketAssetRate, 2)
```

Comment:
These were rounded to integers to avoid the default order price being the same as current price, which is an invalid input for limit and stop orders. It wasn't an ideal solution but worked for the most part. Having it at 2 decimal places increases the chances the price will be the same when rounded as prices are often like `1556.43000000`. I think a better solution is to also add or subtract $0.01 to ensure they are offset from current price.

Reply:
But it doesn't makes sense if we've any price < 1$, right?

Reply:
100.0000 for example just becomes 100.01 or 99.99

Reply:
@avclarke i get you, but does it makes sense if we've any price <= 1$, then it becomes 0.9 or 1.1, that is not correct right?

Reply:
@platschi wdyt ?

Reply:
Sorry I read that as > 1 fist time round. So when price is smaller than 1 say 0.9 then our order price would default to 0.89 or 0.91. It does become a problem when price is 0.01 as we cannot have 0.00. For this case we could try changing the decimal amount. There is a function in number.js called `suggestedDecimals` which we can use to determine the decimal places then always just increment or decrement the last digit to shift the price.

Reply:
@avclarke i think we're already doing it in `floorNumber` and `ceilNumber` functions

<img width="698" alt="Screenshot 2022-10-27 at 12 02 21 AM" src="https://user-images.githubusercontent.com/55599878/198107698-92b124d3-dee2-4cac-ae61-00fb2c3c265e.png">

Reply:
ok perfect, we can not pass any decimals and just increment last digit of what's returned there?

Reply:
It's doing the same what you've just mentioned above.

Reply:
A rough idea of what I'm thinking would be:
```
const decimals = suggestedDecimals(marketAssetRate)
const offset = 1 / 10 ** (decimals ?? 1)
let adjustedPrice = down ? floorNumber(marketAssetRate) : ceilNumber(marketAssetRate)
adjustedPrice = down ? adjustedPrice - offset : adjustedPrice + offset
```

Reply:
@avclarke i've made the changes as you requested.